functions/text-transform/1.0/index.js

```diff
@@ -0,0 +1,31 @@
+const textTransform = async ({
+  value,
+  downcase,
+  upcase,
+  capitalize,
+  parameterize,
+}) => {
+  const transformedValue = parameterize
+    ? parameterized(value)
+    : downcase
+    ? value.toLowerCase()
+    : upcase
+    ? value.toUpperCase()
+    : capitalize
+    ? capitalized(value)
+    : value;
```

Comment:
If we go with one `transformation` option we could use a `switch` statement like
```javascript
const textTransform = async ({ value, transformation }) => {
  let result;

  switch (transformation) {
    case 'downcase':
      result = value.toLowerCase();
      break;
    ...
    default:
      result = value;
  }

  return {
    result,
  };
};
```